vars/RedHat-7.yml

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: MIT
+---
+# Put internal variables here with Red Hat Enterprise Linux 7 specific values.
+
+__certificate_packages:
+  - python-cryptography
+  - python-dbus
```

Comment:
`__certificate_default_directory: /etc/pki/tls` is missing here, I think.

Reply:
Amended.